src/converter-node.ts

```diff
@@ -42,19 +42,16 @@ function convertFileSync(file: string, customArgs?: Arguments): unknown {
 
     try {
         presetBlob = readFileSync(file);
-        presetName = (typeof args.name !== 'undefined' && args.name.trim().length > 0) ? args.name : basename(file, extname(file));
+        presetName = typeof args.name !== 'undefined' && args.name.trim().length > 0 ? args.name : basename(file, extname(file));
         presetDate = args.noDate ? undefined : statSync(file).mtime.toISOString();
         presetObj = convertBlob(presetBlob, presetName, presetDate, args);
     } catch (error) {
         Log.error(error);
     }
 
-    const whitespace: number = (args.minify === true) ? 0 : 4;
+    const whitespace: number = args.minify === true ? 0 : 4;
 
     return JSON.stringify(presetObj, null, whitespace);
 }
 
-export {
-    convertFile,
-    convertFileSync
-};
+export { convertFile, convertFileSync };
```

Comment:
I prefer having these kinds of lists like before, one line at a time -- no matter how short.

I don't feel too strongly about this, so if it's any hassle, or a strong preference of yours, keep it.

Reply:
Actually, I do agree. The problem is that the only wide-used formatter for JavaScript/TypeScript that I'm aware of is [`prettier`](https://prettier.io/), which is opinionated. The formatting of this particular example depends on the print-width. I find it more readable as it was before, but there is no way to disable single rules in `prettier`. The only other way is to set a `prettier-ignore` command – which I find far from ideal.

Reply:
Ah you‘re using a formatter, even better if it‘s opinionated. Then I‘m all for whatever the tool suggests 🙂 Consistency is king.